doc/man3/TS_VERIFY_CTX_set_certs.pod

```diff
@@ -0,0 +1,44 @@
+=pod
+
+=head1 NAME
+
+TS_VERIFY_CTX_set_certs - set certificates for TS reponse verfication
+
+=head1 SYNOPSIS
+
+ #include <openssl/ts.h>
+
+ STACK_OF(X509) *TS_VERIFY_CTX_set_certs(TS_VERIFY_CTX *ctx,
+                                         STACK_OF(X509) *certs);
+
+=head1 DESCRIPTION
+
+The Time-Stamp protocol (TSP) is defined by RFC 3161. TSP is a protocol used to
+provide long term proof of the existence of a certain datum before a particular
+time. TSP defines a Time Stamping Authority (TSA) and an entity who shall make
+requests to the TSA. Usually the TSA is deemed as the server side and the
+requesting entity is deemed as the client.
+
+In TSP, when a server is sending a response to a client, the server normally
+needs to sign the response data - the TimeStampToken (TST) - with its private
+key. Then the client shall verify the received TST by server's certificate chain.
+
+TS_VERIFY_CTX_set_certs() is used to set the server's certificate chain when
+verifying a TST. B<ctx> is the verification context created in advance and
+B<certs> is a stack of B<X509> certificates in PEM format.
```

Comment:
"~~in PEM format~~."  (The `X509` struct contains an internal representation of the certificate)

Reply:
Hmmm, right. I was originally thinking the certificates in file should be in PEM format. But that's not this API's business to care about...